testsuite/tests/lib-stdlib/pervasives_deprecated.ml

```diff
@@ -0,0 +1,32 @@
+(* TEST
+   * expect
+*)
+
+[@@@warning "@A"];;
+
+Pervasives.(+) 1 1;;
+[%%expect{|
+Line 3, characters 0-14:
+  Pervasives.(+) 1 1;;
+  ^^^^^^^^^^^^^^
+Error (warning 3): deprecated: module Stdlib.Pervasives
+Use Stdlib instead
```

Comment:
The test needs to be updated due to the final dot after `instead`.

Reply:
yep, just saw that